packages/documentation/src/pages/getting-started/common-tasks/reviewing-feature-branches.mdx

```diff
@@ -1,51 +0,0 @@
-# Reviewing changes in feature branches
```

Comment:
Consolidated this into the "Review" document. Felt like it was easier to explain why these exist if their explanation is in the same context as our typical workflow